docsrc/sasl/authentication_mechanisms.rst

```diff
@@ -10,35 +10,50 @@ Mechanisms
 ANONYMOUS
 ---------
 
-.. todo::
-   Content needed here
+As used in FTP this mechanism only passes an email address for authentication.
```

Comment:
This is unclear if this is true or false. The rfc talks about an email. Can somebody check the code and confirm ?